File: Assets/GraphQL/Model/Points/Point.cs
using GraphQL.Helpers;

namespace GraphQL.Model {
	public class Point {
		[GqlSelection("pointID")] public int PointID { get; private set; }
		[GqlSelection("pointTypeID")] public int PointTypeID { get; private set; }
		[GqlSelection("cityID")] public int CityID { get; private set; }
		[GqlSelection("name")] public string Name { get; private set; }
		[GqlSelection("address")] public string Address { get; private set; }
		[GqlSelection("price")] public float Price { get; private set; }
	}
}

File: Server/src/schema/Points/point.ts
import { Field, Float, InputType, Int, ObjectType } from "type-graphql";
import { BaseEntity, Column, Entity, PrimaryGeneratedColumn } from "typeorm";

@Entity()
@ObjectType()
export default class Point extends BaseEntity {
    @Field(() => Int!)
    @PrimaryGeneratedColumn()
    pointID!: number;

    @Field(() => Int!)
    @Column()
    pointTypeID!: number;

    @Field(() => Int!)
    @Column()
    cityID!: number;

    @Field(() => String!)
    @Column()
    name!: string;

    @Field(() => String!)
    @Column()
    address!: string;

    @Field(() => Float!)
    @Column()
    price!: number;

    constructor() {
        super();

        this.pointID = 0;
        this.pointTypeID = 0;
        this.cityID = 0;
        this.name = "None";
        this.address = "None";
        this.price = 0;
    }
}